misc/help.talon

```diff
@@ -1,4 +1,18 @@
 help alphabet: user.help_alphabet(user.get_alphabet())
+help symbols: user.help_symbol_key_words(user.get_symbol_key_words())
+help numbers: user.help_number_key_words(user.get_number_key_words())
+help punctuation: user.help_punctuation_words(user.get_punctuation_words())
+help modifier: user.help_modifier_words(user.get_modifier_words())
+help simple keys: user.help_simple_keys(user.get_simple_keys())
```

Comment:
I wonder if simple keys, alternate keys, f keys, and arrows could be merged to 'help special keys'? Would people want to refer to them distinctly?